VideoApi/VideoApi.Domain/ParticipantStatus.cs

```diff
@@ -12,7 +12,9 @@ public ParticipantStatus(ParticipantState participantState)
             TimeStamp = DateTime.UtcNow;
         }
 
+        public Guid ParticipantId { get; set; }
+        public ParticipantBase Participant { get; set; }
```

Comment:
Why is participant a property of ParticipantStatus? ParticipantBase also has a list of ParticipantStatus